### DISCUSSION TITLE
能否根据程序来分流？

### DISCUSSION THREAD

**Discussion by YUUUCC** [IrineSistiana/mosdns#859] [Q&A]
### 问题阐释
有没有什么办法实现根据程序来进行分流，比如程序A发起的所有DNS查询请求都转发到远程DNS。

**Comment by sam452167735789** [comment]
使用特定组运行应用，或者使用cgroups控制进程分组，再使用iptables的-m owner --uid-owner myappuser -j MARK来实现。
